utils/tagged_integer.hh

```diff
@@ -10,6 +10,7 @@
 
 #include <cstdint>
 #include <compare>
+#include <fmt/core.h>
 #include <iostream>
 #include <type_traits>
```

Comment:
could you put this include under `<type_traits>` ? so we can group the standard headers together? we don't have a policy for the order, but it would be ideal if we can make this more ordered.

Reply:
Sure, fixed!